### PR TITLE
Skip db_sync run on SUSE

### PR DESCRIPTION
**Pull request by dirkmueller**
It is already run by the init script, and since it runs
with the wrong user, it screws everything up.

### CODE REVIEW

File: chef/cookbooks/openstack-database/recipes/api.rb [hunk 1]
```diff
@@ -96,5 +96,8 @@ class ::Chef::Recipe # rubocop:disable Documentation
 end
 
 execute 'trove-manage db_sync' do
+  user  node['openstack']['database']['user']
+  group node['openstack']['database']['group']
+
   notifies :restart, 'service[trove-api]', :immediately
```
Comment (dirkmueller) [inline chef/cookbooks/openstack-database/recipes/api.rb]: I don't think so.. but this is unrelated to this git commit ;-)

**Comment by vuntz** [comment]
+1

**Comment by nkrinner** [comment]
+1
